packages/graphiql/src/components/QueryHistory.js

```diff
@@ -114,6 +89,22 @@ export class QueryHistory extends React.Component {
     );
   }
 
+  updateHistory = (query, variables, operationName) => {
```

Comment:
I noticed in other areas that public methods were marked with a `Public API` comment which I didn't add here. Could you add it in for me?

e.g.:
https://github.com/graphql/graphiql/blob/e765543a9454c87a213ec60466341fa12cce1ab9/packages/graphiql/src/components/DocExplorer.js#L147

Reply:
sure yes! we are about to deprecate this “plugin api” possibly

Reply:
yeah I figured this would go away once we moved to more React 16 patterns anyways but good to make it clear and consistent until then

Reply:
yep! this reminded me i need to add a deprecation notice